utils/images.ahk

```diff
@@ -27,3 +27,9 @@ GetCurrentMapPage() {
         }
     }
 }
+
+SearchUpgrade(path) {
+	if (SearchImage("buttons\upgrades\1R", "", 250, (325+path*150), 1494, (349+path*150)) or SearchImage("buttons\upgrades\1L", "", 250, (325+path*150), 1494, (349+path*150))) {
```

Comment:
`"buttons\upgrades\1R",` should probably be `"buttons\upgrades\" path "R",` and same for the `L` version